pkgs/development/ocaml-modules/ocamlearlybird/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildDunePackage, angstrom, angstrom-lwt-unix,
+  batteries, cmdliner, lwt_ppx, ocaml_lwt, ppx_deriving_yojson,
+  ppx_tools_versioned, yojson }:
+
+buildDunePackage rec {
+  pname = "earlybird";
+  version = "0.1.5";
+
+  minimumOCamlVersion = "4.04";
+
+  src = fetchFromGitHub {
```

Comment:
Would it be better to fetch the release asset from <https://github.com/hackwaly/ocamlearlybird/releases/download/0.1.5/earlybird-0.1.5.tbz>?

Reply:
When using the released source archive, `ocamlearlybird --version` returns “0.1.5” rather than “%%VERSION%%”.

Reply:
Done.